codegen.yml

```diff
@@ -15,3 +15,9 @@ generates:
       baseTypesPath: types/graphql.ts
     plugins:
       - typescript-operations
+
+  src/possibleTypes.json:
```

Comment:
Can we move this to be in the same directory as `graphql.ts` (types/*)

Reply:
Yeah, totally. I just wanted to get it done really. Will fix tomorrow

Reply:
Fixed